client/components/jetpack/licensing-prompt-dialog/index.tsx

```diff
@@ -56,9 +61,43 @@ function LicensingPromptDialog( { siteId }: Props ) {
 						} )
 				  )
 				: preventWidows( translate( 'Your product is pending activation' ) );
+		} else {
+			titleToRender = preventWidows( translate( 'Activate your new Jetpack features' ) );
 		}
-		return preventWidows( translate( 'Activate your new Jetpack features' ) );
-	}, [ detachedUserLicense, hasOneDetachedLicense, translate ] );
+	}
+
+	let instructions: React.ReactNode = preventWidows(
+		translate(
+			'Find the license key in your purchase confirmation email to activate your new Jetpack features.'
+		)
+	);
+
+	if ( isEnabled( 'jetpack/pricing-page-rework-v1' ) ) {
```

Comment:
Hmm, I though on a meeting we wanted it to be used for current version as well, not under feature flag, though, I might be mistaken

Reply:
Bryan in the Asana task mentioned that we want it to be behind feature flag. Otherwise, it would have been easier to deal with it without feature-flag.